docs(tabs-overflow): update a11y docs (#5621)
adobe/spectrum-web-components: packages/tabs/tabs-overflow.md

## Overview

> The `<sp-tabs-overflow>` is a decorative component that works in conjunction with the `<sp-tabs>` component. It enables horizontal tab scrolling in cases where the available width is insufficient to display all tabs. Do not truncate multiple tab items just to make them fit horizontally.

Reviewer: ```suggestion
The `<sp-tabs-overflow>` is a decorative component that works in conjunction with the `<sp-tabs>` component. It enables horizontal tab scrolling in cases where the available width is insufficient to display all tabs. Multiple tab items should not be truncated solely to accommodate horizontal layout constraints.
```

_Note: Vertical scrolling is not yet supported._

[View the design documentation for this component.](https://spectrum.adobe.com/page/tabs/)

### Usage

[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/tabs?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/tabs)
[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/tabs?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/tabs)

```zsh
yarn add @spectrum-web-components/tabs
```

To import the `<sp-tabs-overflow>` component with its associated side effects, use the following import statement:

> ```ts

Reviewer: ```suggestion
```js
```

import '@spectrum-web-components/tabs/sp-tabs-overflow.js';
```

To utilize the `TabsOverflow` base class as a type or for extending its functionality, utilize the following method:

> ```ts

Reviewer: ```suggestion
```js
```
